### PR TITLE
issue 22: implement FirebaseClient, QuitGame, fix cell paths

### PR DESCRIPTION
**Pull request by saviotsui**
I'm going to split this Issue into a couple PRs.

Basically all I did here was hook up the Quit Button.  The firebase client has the query to take care of this removal from the db table here https://console.firebase.google.com/project/mindgame-91529/database/data/user_in_game

### CODE REVIEW

File: ios-trivia-game/API/FirebaseClient.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+class FirebaseClient {
+    static let instance = FirebaseClient()
+    
+    private init() {}
+    
+    func getUser(userId: String, complete: @escaping (FIRDataSnapshot) -> (), onError: ((Error?) -> ())?) {
+        let ref = FIRDatabase.database().reference()
+        ref.child("\(Constants.USER_TABLE_NAME)/\(userId)").observe(.value, with: { (snapshot) in
+            Logger.instance.log(logLevel: .info, message: "Table: \(Constants.USER_TABLE_NAME), Removing: \(User.currentUser?.uid)")
```
Comment (zhiachong) [inline ios-trivia-game/API/FirebaseClient.swift]: Is this function supposed to "remove" something?

File: ios-trivia-game/API/FirebaseClient.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            if (onError != nil) {
+                onError!(error)
+            }
+        }
+    }
+    
+    // quits game of current user
+    func quitGame(complete: (() -> ())?, onError: ((Error?) -> ())?) {
+        let currentUserId = User.currentUser?.uid
+        let ref = FIRDatabase.database().reference()
```
Comment (zhiachong) [inline ios-trivia-game/API/FirebaseClient.swift]: This can be a property.

File: ios-trivia-game/API/FirebaseClient.swift (hunk 1 above)
Comment (saviotsui) [reply ios-trivia-game/API/FirebaseClient.swift]: Oops. I will update the log statement

File: ios-trivia-game/API/FirebaseClient.swift (hunk 2 above)
Comment (saviotsui) [reply ios-trivia-game/API/FirebaseClient.swift]: Is it safe to leave this ref around for the duration of app's lifetime?

**Review by zhiachong — COMMENTED** [review]
overall looks good, just some changes.

**Pull request merged**